packages/mobile/src/screens/wallet-connect/usePhantomConnect.ts

```diff
@@ -4,25 +4,42 @@ import { tokenDashboardPageActions } from '@audius/common/store'
 import { useRoute } from '@react-navigation/native'
 import { useDispatch } from 'react-redux'
 
-import { connectNewWallet, signMessage } from 'app/store/wallet-connect/slice'
+import {
+  connect,
+  connectNewWallet,
+  signMessage
+} from 'app/store/wallet-connect/slice'
 
 import type { WalletConnectRoute } from './types'
 
 const { updateWalletError } = tokenDashboardPageActions
 
 export const usePhantomConnect = () => {
   const dispatch = useDispatch()
-  const { params } = useRoute<WalletConnectRoute<'Wallets'>>()
+  const route = useRoute<WalletConnectRoute<'Wallets'>>()
+  // I have no idea what's going on here. I don't think this works currently,
```

Comment:
fr no cap what is happening??? this change makes it work... maybe a local dev thing?